### ISSUE TITLE
Seeming observation ID and Community Taxon ID bug or bug-like issue

### ISSUE THREAD

**Issue by bdagley** [inaturalist/inaturalist#4169]
Currently, an observer identified species Bombus (subgenus Pyrobombus) vagans, then someone made a disagreeing genus Bombus ID, then 4 people made a species Bombus (subgenus Pyrobombus) bimaculatus ID, but the observation ID and the community taxon (CID) is still subgenus Pyrobombus, non-research grade. All 6 IDs include genus Bombus. This seems to suggest that the disagreeing ID is also counting as disagreeing with subgenus Pyrobombus and/or genus Bombus, but would seem more accurate if it were to only count as disagreeing with species B. vagans (as the ID itself says), not involving the subgenus or genus. This seems to be a bug, bug-like issue, or at minimum less accurate than it could be. 

I think a similar issue, or this identical one, has been raised by various people in the past on iNaturalist, and possibly on here. In the event that this identical issue is already in this github, I'd be open to merge/link the current Issue with it, but would also like some feedback on this. If I had to guess, although I don't have the full information to know, I wonder if this could be due to the fact that the infraranks (including subgenus, vs. only having genus and species ranks) were added to iNaturalist after the time that the community taxon algorithm and website were originally created, as far as I know. 

https://www.inaturalist.org/observations/88534033

**Comment by bdagley** [comment]
https://www.inaturalist.org/observations/88534033

**Comment by pleary** [comment]
> the disagreeing ID is also counting as disagreeing with subgenus Pyrobombus

This is correct. This is the intentional behavior and not a bug. An additional ID has since been added and the observation is now research grade at species

**Comment by bdagley** [comment]
@pleary This is again misleading to simply close as "intentional, not a bug," because that ignores the bug-like issue present here. First, it's irrelevant that an additional ID has been added and changed the observation. The example I referred to is for the obs. before an additional ID was added. 

To further clarify the issue, an ID disagreeing with Bombus (subgenus Pyrobombus) vagans shouldn't count as disagreeing with subgenus Pyrobombus, because in principle an identifier could disagree with a species but think an observation is a different species of the same subgenus. Furthermore, 4 people in the example had added the correct species ID (of another Pyrobombus species), which no one disagreed with, yet the CID still hadn't become species rank. The disagreeing ID for the species also shouldn't disagree with the genus Bombus (in the event that it does), given that it is itself a Bombus ID, although I'm unsure if it only disagrees with subgenus. Anyway, like the CID not transferring to GBIF issue, this issue involves what the iNat webpage explicitly states being inaccurate. Because the disagreeing ID says "[user] disagrees this is Bombus vagans," not that they disagree with the subgenus, which isn't implied. I also asked if infraranks, including subgenera, were only added to iNat after the original design of the observation algorithm. Because that could explain how/why this is occurring, and importantly would mean that the behavior wasn't intended (at least not when the system was originally designed).

I've only made a few iNat issues on GitHub, and only do so rarely. Although, it seems fair to include instances like these where the webpage is stating something that the website isn't doing or is doing the opposite of, whether bugs or not, and I believe the staff and other devs have at times previously made issues like that. They're still instances in need of correction. And the problem with closing these issues is that they then might not be considered or remembered. @kueda @loarie @tiwane

**Comment by pleary** [comment]
As we say in the [Contributing guide](https://github.com/inaturalist/inaturalist/blob/main/CONTRIBUTING.md) for this Github repository:

> We mostly use Github for managing implementation, not for discussing what to implement or why we might want to implement it

It sounds like you'd like to propose changes to existing functionality. For topics like that, please use the [iNaturalist Forum](https://forum.inaturalist.org/) where the iNaturalist community can contribute to the discussion. If the discussion leads to change we intend to make, someone on the iNaturalist staff will create a Github ticket with implementation requirements.

**Comment by bdagley** [comment]
I explained the matter in each of these issues in detail. The obs. CID not being sent to GBIF (i.e., that the obs. ID is really what becomes RG, not the CID, contrary to the phrasing of the DQA "can the CID be improved?" question), and the current taxon ID issue, are issues where the website's description of it's own functionality is misleading or inaccurate. I remember seeing past issues here where staff, mods, or devs raised similar matters or spoke in ways like "if the website implies X occurs but it doesn't, should we change that?" Furthermore, what staff have commented on the CID not being sent to GBIF issue when others raised it seems to be agreement. Re: the Forum, the posts (and private emails) by myself and some others tended to be ignored by staff and devs, and rarely implemented. And as you linked to, a mod already created a similar Forum issue back in 2019, which staff/admin did reply to. So, it seems, and many others agree or would agree with me on this, that I have submitted helpful documentation here, which should be considered by all of the staff and devs, even if you close some of the issues. 

You also seem to have kept open the species complex CIDs being sent to GBIF as species IDs-issue. There, I wonder what's the difference that makes that one a true bug. I assume, because a species record can be sent in that way even when no one made a species ID on the obs. (e.g., they only made complex IDs). Yet, I consider the other issue of the CID not being sent to GBIF in general very similar, even overlapping, and the current issue here also involved infraranks (some of the problems seem to be due to the addition of infraranks after the original design of the website). Bear in mind that it also takes a long time for people (not limited to me) to notice, document, and post these issues, only for them to seem to be dismissed. That is my input, and I won't be using the Forum, so I'll leave it at that.
